src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -877,6 +922,20 @@ export class PatientViewPageStore {
         []
     );
 
+    readonly molecularData = remoteData<NumericGeneMolecularData[]>(
```

Comment:
seems odd that this is called molecularData (general) when it derives from discreteCnaData?

Reply:
This is to make the cna data compatible with existing utility functions which accept molecular data only. We should probably refactor the store to use the molecular data endpoint to fetch cna data (if possible) to avoid this type of hacky data conversion.

Reply:
seems like we should have a comment explaining it.  or maybe just called discreteCNAMolecularData?